model/hardware/registry.yaml

```diff
@@ -92,18 +92,520 @@ groups:
+      - id: hw.network.model
+        type: string
+        stability: development
+        brief: >
+          Descriptive model name of the network adapter
+        examples: ['Intel X520', 'Mellanox ConnectX-5']
+      - id: hw.network.logical_addresses
+        type: string[]
+        stability: development
+        brief: >
+          Logical addresses of the adapter (e.g. IP address, or WWPN)
+        examples: [['172.16.8.21', '57.11.193.42']]
+      - id: hw.network.physical_address
+        type: string
+        stability: development
+        brief: >
+          Physical address of the adapter (e.g. MAC address, or WWNN)
+        examples: ['00-90-F5-E9-7B-36']
+      - id: hw.network.serial_number
+        type: string
+        stability: development
+        brief: >
+          Serial number of the network adapter
+        examples: ['ABCDEF123456']
+      - id: hw.network.vendor
+        type: string
+        stability: development
+        brief: >
+          Vendor name of the network adapter
+        examples: ['Intel', 'Broadcom', 'Mellanox']
+      - id: hw.network.direction
+        type:
+          members:
+            - id: receive
+              value: 'receive'
+              brief: "Receive"
+              stability: development
+            - id: transmit
+              value: 'transmit'
+              brief: 'Transmit'
+              stability: development
+        stability: development
+        brief: >
+          Direction of network traffic
+
+      # PHYSICAL DISK ATTRIBUTES
+      - id: hw.physical_disk.firmware_version
+        type: string
+        stability: development
+        brief: >
+          Firmware version of the physical disk
+        examples: ['2.3.1']
+      - id: hw.physical_disk.model
+        type: string
+        stability: development
+        brief: >
+          Descriptive model name of the physical disk
+        examples: ['Samsung 970 EVO Plus', 'WD Black SN750']
+      - id: hw.physical_disk.serial_number
+        type: string
+        stability: development
+        brief: >
+          Serial number of the physical disk
+        examples: ['S3Z9NX0M123456']
+      - id: hw.physical_disk.type
+        type: string
+        stability: development
+        brief: >
+          Type of the physical disk
+        examples: ['HDD', 'SSD', '10K']
+      - id: hw.physical_disk.vendor
+        type: string
+        stability: development
+        brief: >
+          Vendor name of the physical disk
+        examples: ['Samsung', 'Western Digital', 'Seagate']
+      - id: hw.physical_disk.state
+        type:
+          members:
+            - id: remaining
+              value: 'remaining'
+              brief: "Remaining"
+              stability: development
+        stability: development
+        brief: >
+          State of the physical disk endurance utilization
+      - id: hw.physical_disk.smart_attribute
+        type:
+          members:
+            - id: raw_read_error_rate
```

Comment:
These are merely examples of SMART attributes. We cannot restrict the value of this attribute.

Reply:
the question still remains open - is it better to leave it as enum or as a sting with just a few examples?

Reply:
Updated